camp_fin/views.py

```diff
@@ -1191,9 +1197,143 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class CandidateDetail(CommitteeDetailBaseView):
+"""
```

Comment:
let's move this stuff into a separate file. we also probably also need to have a copy of the django-extensions License in that file.